src/system-health/health_checker/config.py

```diff
@@ -0,0 +1,140 @@
+import os
+import json
+from sonic_py_common import device_info

```

Comment:
Group imports per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports), and alphabetize within groups:

```suggestion
import json
import os

from sonic_py_common import device_info
```

Reply:
Hi, thanks for sharing the standard. I will fix them soon.